### ISSUE TITLE
Links on the website don't have any focus indicator

### ISSUE THREAD

**Issue by stof** [async-aws/aws#410]
They are intentionally removed at https://github.com/async-aws/aws/blob/b567df8e1ad1c681da0f5c99815e282a4b8e23e2/website/assets/style/main.scss#L26-L30

This breaks accessibility.

**Comment by Nyholm** [comment]
Closed by #420. 

Thank you for reporting this. It is very much appreciated.

**Comment by Nyholm** [comment]
> The only remaining issues found by the Firefox accessibility dev tools are about the link on the logo in the sidebar (the logo has no alt attribute and the link has no label, meaning that the link is not accessible)
